### PR TITLE
Refactor to use npm scripts and serve-static, update deps

### PR DESCRIPTION
**Pull request by ashleygwilliams**
in reference to #99 

 #97 is still an issue, will fix in another PR, this does move us closer to detangling that tho

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 module.exports = function(grunt) {
+  require('load-grunt-tasks')(grunt);
```
Comment (jmeas) [inline Gruntfile.js]: :heart_eyes:

File: bin/serve [hunk 2]
```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env node
-server = require('../server/server.js');
-
-server({ port : 4444, dev : true });
```
Comment (jmeas) [inline bin/serve]: :wave:

File: help.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-To work on the tests, open a browser and visit http://localhost:{{port}}.
+To work on the tests, open a browser and visit http://{{host}}:{{port}}.
```
Comment (jmeas) [inline help.txt]: Where do `{{host}}` / `{{port}}` come from?

Comment (jmeas) [inline help.txt]: nvm. I see it.

Comment (jmeas) [inline help.txt]: Do you think adding the defaults here, and then also how to override them, is a good idea?

Comment (ashleygwilliams) [inline help.txt]: https://github.com/rmurphey/js-assessment/blob/refactor/index.js#L6-L7
https://github.com/rmurphey/js-assessment/blob/refactor/index.js#L6-L7

**Pull request merged**